docs/development/config-files.md

```diff
@@ -197,3 +197,18 @@ In order to get your configuration file included in this library, two things mus
 1. Update the index file using `yarn run config import -i`.
 1. Check your new or changed files for potential problems using `lerna run lint:config`. Warnings in your file may be tolerated if there is a good reason for them. Errors must be fixed.
 1. Create a PR.
+
+If your device is already present in [OZW devices DB](https://github.com/OpenZWave/open-zwave/tree/master/config), you can import it by using its device id (you can find it in zwavejs2mqtt control panel) with this command: 
+
+```bash
+npm run config -- import -s ozw -Dmid --ids 0x0086-0x0075-0x0004
```

Comment:
the `i` parameter was removed, we auto-update the index.json now.
Also there's no need to always update `manufacturers.json`, so we should probably document that separately and remove the `m` parameter here too.

Reply:
We should also mention the `yarn` command, which is very similar:
```
npm run config -- import -s ozw -Dd --ids 0x0086-0x0075-0x0004
yarn run config import -s ozw -Dd --ids 0x0086-0x0075-0x0004
```